cmd/minishift/cmd/start.go

```diff
@@ -555,6 +555,7 @@ func determineIsoUrl(iso string) string {
 	case minishiftConstants.CentOsIsoAlias:
 		iso = constants.DefaultCentOsIsoUrl
 	case minishiftConstants.MinikubeIsoAlias:
+		fmt.Printf("\n\nNote: minikube ISO will be depericated in next release, please use b2d or centos ISO.\n\n")
```

Comment:
s/depericated/depricated

Trying installing a dictionary plugin in Sublime Text and turn on spellcheck.